virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -160,6 +160,16 @@ def create_disk(
         )
 
 
+def _all_disk_targets():
```

Comment:
Please use `def get_all_disks_target()` . The function name starts with a verb to align with other functions. And not use `_ `at the beginning